encrypt.js

```diff
@@ -0,0 +1,27 @@
+const hkdf = require('futoin-hkdf');
```

Comment:
I'm wondering how we can make this task as developer-friendly as possible. I think we might be able to make an improvement here. For example I'm wondering:

- If we keep encrypt as a module, is there a reason this file isn't in the cypress dir?
- If we want an alternative method, rather than a module, for defining a custom encrypt task, is there a place in config the developer could set the custom encrypt function?
- If the developer wants to overwrite the encrypt task, couldn't they just overwrite in Cypress code?

So I guess my question to you is: do you think a standalone module is the most developer-friendly way to define the encrypt task?

Reply:
Additionally, if we use this approach of having encrypt be a module, can we at least avoid requiring the developer to register the task? I'd like to have as much work 'out of the box' as possible given the complexity involved in the developer setting up a tenant and user for testing.

Reply:
Hi @sir-dunxalot - thanks for taking a look at this.

I think the most developer friendly way to do this would be in the browser, I just couldn't find a browser solution when having a quick look. If I have some time this week - I'll see if I can find a way to do this on the FE and remove the need to register a task.

Reply:
@adamjmcgrath Were you able to investigate this?

Reply:
Hey @sir-dunxalot - I had a look at this on Friday and couldn't bend WebCrypto to my will, so had to leave it